### ISSUE TITLE
Field Transfer Request for 3rd party extension

### ISSUE THREAD

**Issue by Brad403** [hougaard/Simple-Object-Designer#338]
Hello,

Is it possible for SOD to add additional field transfer for 3rd party extensions? I have been using SOD to add in additional fields for a 3rd party extension called "Rental Process Management" and while SOD has no problem creating these fields for the tables related to RPM, I have noticed there is no available fields when trying to use the "Field Transfer".

Thanks,

Brad

**Comment by hougaard** [comment]
No... The field transfer section is me "teaching" the Simple Object Designer how to do this. Every one of the 127 current field transfers is me painstakingly figuring out the events to subscribe to, how the parameters need to be passed, and so on...

Sorry :(
